introduction-to-datasets.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: page
+title: Introduction to Datasets
+---
+
+A dataset is a collection of records, similar to a relational database table. Records are similar to table rows, but the columns can contain not only strings or numbers, but also nested data structures such as lists, maps, and other records.
+
+To define a dataset, Kite minimally requires a [URI](#uris) and a [schema](#schemas).
+
+## URIs
+
+Kite identifies datasets by URI. The URI you provide tells Kite how and where to store data. For example, a dataset created with the URI `dataset:hdfs:/user/cloudera/datasets/movies` is stored in HDFS.
```

Comment:
I think it would be valuable to say that URIs provide a few things:
1. The underlying storage (which you already cover here) based on the scheme
2. A name and a namespace. These are used when the underlying system has the same concept (as with Hive) and can also be listed using the [Datasets.list method](http://kitesdk.org/docs/0.17.1/apidocs/org/kitesdk/data/Datasets.html#list(java.lang.String)).
3. Sometimes, a location that could be a HDFS path

This would help later when you note that the namespace and name components have restrictions on them.